include/picongpu/particles/Particles.hpp

```diff
@@ -249,14 +249,6 @@ namespace picongpu
         }
     };
 
-    namespace traits
```

Comment:
Since I was required to refactor the included order to avoid cyclic dependencies this shows up as unused stuff that introduces include issues. Moving this into a separate PR will introduce no advantage.